packages/template-retail-react-app/app/components/_app/index.test.js

```diff
@@ -16,34 +16,23 @@ import {DEFAULT_LOCALE} from '@salesforce/retail-react-app/app/utils/test-utils'
 import useMultiSite from '@salesforce/retail-react-app/app/hooks/use-multi-site'
 import messages from '@salesforce/retail-react-app/app/static/translations/compiled/en-GB.json'
 import mockConfig from '@salesforce/retail-react-app/config/mocks/default'
-import * as constants from '@salesforce/retail-react-app/app/constants'
+import {prependHandlersToServer} from '@salesforce/retail-react-app/jest-setup'
+import {mockCustomerBaskets} from '@salesforce/retail-react-app/app/mocks/mock-data'
 
 jest.mock('../../hooks/use-multi-site', () => jest.fn())
 jest.mock('../../hooks/use-update-shopper-context', () => ({
     useUpdateShopperContext: jest.fn()
 }))
 
 let windowSpy
-let originalValue
-beforeAll(() => {
-    jest.spyOn(console, 'log').mockImplementation(jest.fn())
```

Comment:
I don't see why we should suppress console.log here

Reply:
Nice, thanks for always leaving the code better than before.